packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/BasicFolderTree.1.doc.mdx

```diff
@@ -1,3 +1,38 @@
 import BasicFolderTree from './BasicFolderTree?dev-site-example';
 
+# Basic Folder Tree
+
+### Description
+
+The following example demonstrates a basic folder tree. For more details on specific folder tree
+features, see the following examples:
+* [Selection](./folder-tree-selection)
+* [Expand/Collapse](./folder-tree-expand-collapse)
+* [Expand/Collapse All](./folder-tree-expand-collapse-all)
+
+A folder tree is composed of two components:
+* **[FolderTree](../about#foldertree-props-table)**: The top-level folder tree component.
+* **[FolderTreeItem](../about#foldertreeitem-props-table)**: A row in a folder tree that may have `subfolderItems`.
```

Comment:
Updated to `FolderTree.Item` since that's how it will be used most of the time. [dd47fb0](https://github.com/cerner/terra-framework/pull/1973/commits/dd47fb0cdd8f5375b8c145e5c19f3ed06b2df5bf)